src/app.py

```diff
@@ -37,5 +37,23 @@ def get_all_task_handler():
     jsonTaskList = json.dumps(taskList, indent = 8)
     return jsonTaskList, HttpCode.Success.value, {'Content-Type': 'application/json'}
 
+@app.route('/user', methods=['GET', 'POST'])
+def user_handler():
+    if request.method == 'POST':
+        if request.headers.get('Content-Type') == 'application/json':
+            return userController.create_user(request)
+        else:
+            return "Invalid Content-Type", HttpCode.BadRequest.value
+    elif request.method == 'GET':
+        userEmail = request.args.get('userEmail',default='',type=str)
```

Comment:
@sweiyang , i use a request param for userEmail is it okay?

Reply:
Yes its ok